docs/glossary.rst

```diff
@@ -478,10 +478,42 @@ defined role within the organization (see our reference on :ref:`user roles
 
     :ref:`User roles <user-roles>`
 
+.. _gloss-version:
+
+Versions
+--------
+
+CrateDB uses a versioning system with three levels of versioning: major
+versions, minor versions, and patch versions. (Versions can also be referred to
+as releases.) These apply also to CrateDB Cloud clusters.
```

Comment:
I am wondering if this line is a little ambiguous and could suggest to some users that we version the CrateDB Cloud software in the same way. Could this offer a bit more clarity?

```suggestion
as releases.) We support upgrading the CrateDB version that your cluster is running .

Reply:
Good point. I'll rephrase that